js/app.js

```diff
@@ -85,16 +85,35 @@ function canFactionBePicked(faction, selectedFactions) {
 
 function randomizeFactions() {
   const availableFactions = Array.from(DATA.FACTION_LIST_BY_REACH);
-  const numPlayers = State.playerList.length;
+  const humanPlayers = State.playerList.length;
+  const useBot = document.getElementById("use-bot")
+
+  if(humanPlayers == 1) {
+    useBot.checked = true;
+  }
+  const numPlayers = (useBot.checked ? humanPlayers+1 : humanPlayers);
+  if(numPlayers == 0) {
+    throw "The game can't play itself!";
+  }
+  else if(numPlayers > availableFactions.length) {
+    throw "Not enough available factions for this player count.";
+  }
+
   const minReach = DATA.REACH_BY_PLAYER_COUNT[numPlayers];
   var currentReach = 0;
   const selectedFactions = [];
 
-  if(numPlayers > availableFactions.length) {
-    throw "Not enough available factions for this player count.";
+  function selectFaction(faction) {
+    selectedFactions.push(faction);
+    availableFactions.splice(availableFactions.indexOf(faction), 1);
+    currentReach += faction.reach;
+  }
+
+  if (useBot.checked) {
+    selectFaction(DATA.FACTIONS.MARQUISE_DE_CAT);
```

Comment:
The bot player object has this data stored into it. Perhaps this scope should:
1. Select a random bot player
2. get the faction from the player's field
rather than hard code a faction

something like:
```javascript
if (useBot.checked) {
    const bot = /* select a bot from DATA.PLAYERS/DATA.BOT_PLAYERS */;
    selectFaction(bot.faction);
}
```